dozer-integrations/dozer-spring-support/dozer-spring4/src/test/resources/applicationContext.xml

```diff
@@ -27,7 +27,7 @@
 
     <!-- Provided Spring Integration -->
     <bean id="byFactory" class="com.github.dozermapper.spring.DozerBeanMapperFactoryBean">
-        <property name="mappingFiles" value="classpath*:mappings/*.xml"/>
+        <property name="mappingFiles" value="classpath*:mappings/mappingSpring.xml"/>
```

Comment:
Any reason for this change? if not, please revert

Reply:
reverted